src/main/java/com/spring/beans/SummerWheel.java

```diff
@@ -0,0 +1,16 @@
+package com.spring.beans;
+
+import org.apache.log4j.Logger;
+import org.springframework.stereotype.Component;
+
+@Component("summerWheel")
```

Comment:
it is default value 
https://stackoverflow.com/questions/48871789/how-are-beans-named-by-default-when-created-with-annotation

Reply:
It doesn`t work, when I delete this line.